### DISCUSSION TITLE
如何通过插件扩展Taro的API。

### DISCUSSION THREAD

**Discussion by victorWuxz** [NervJS/taro#13368] [Q & A] [answered]
我们这次开发的车载小程序，基于腾讯车联平台，底层跟wx小程序一致，但相对于wx小程序的API有所扩展，比如wx.getCarInfo()，wx.getCarSpeed等汽车专有的API，我们可以通过编写taro插件扩展wx和taro的api吗，官网没看到相关的文档

**Comment by AdvancedCat** [comment, marked as answer]
如果“车载小程序”比较wx小程序有较多差异，那么建议你以编写纵向小程序端平台插件来支持，具体文档：https://docs.taro.zone/docs/platform-plugin/

如果差异较小，则可以使用 [`@tarojs/plugin-inject`](https://www.npmjs.com/package/@tarojs/plugin-inject) 插件来支持扩展
